networkx/algorithms/cluster.py

```diff
@@ -15,6 +16,7 @@
 ]
 
 
+@nx._dispatch("triangles")
```

Comment:
This was added for test coverage, because we don't actually have a need for explicit names in the current dispatchable algorithms, but we know we will need them in the future.